### PR TITLE
Missing lines in the French translation

### PR DESCRIPTION
**Pull request by nclm**

### CODE REVIEW

File: app/assets/javascripts/components/locales/fr.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const fr = {
   "tabs_bar.public": "Public",
   "tabs_bar.notifications": "Notifications",
   "compose_form.placeholder": "Qu’avez vous en tête&nbsp;?",
-  "compose_form.publish": "Publier",
+  "compose_form.publish": "Pouet",
```
Comment (Gargron) [inline app/assets/javascripts/components/locales/fr.jsx]: Is this French for "toot" or?

**Comment by Gargron** [comment]
Thank you this is great since automated tests were failing because of missing locale keys.

File: app/assets/javascripts/components/locales/fr.jsx (hunk 1 above)
Comment (nclm) [inline app/assets/javascripts/components/locales/fr.jsx]: Yes!!! Pierrec [thought of it](https://mastodon.social/users/Pierrec/updates/100589) :)

**Comment by nclm** [comment]
@Gargron Visibly there’s still an issue somewhere, according to the live website? There’s a lot of messages like this in the console when the settings are set to French:

`[React Intl] Missing message: "compose_form.publish" for locale: "fr", using default message as fallback.`

The preferences section and individual profiles/toots pages are fine.

**Comment by Gargron** [comment]
We have tests for missing locale keys server-side, but nothing for the client-side sadly. I only recently added i18n at all so I don't know yet how to manage those efficiently.

**Comment by nclm** [comment]
Okay. Is this normal still that the main website appears in English even after choosing French? (maybe the updates are just not deployed yet, in this case ignore this and I’ll wait)

**Pull request merged**